### DISCUSSION TITLE
Help or Discord Community?

### DISCUSSION THREAD

**Discussion by lyrx2k** [FerretDB/FerretDB#4581] [General]
Hey guys im using Mongo Atlas, i wanted to use Ferret but idk there are some issues with the setup and actually working ..

is there a discord community or else to seek help ? ..

Im gonna scrap the hole install and will try to redo it with help..

### What exactly do I want to achieve?
- I would like to use ferret as an alternative to mongo atlas (preferably without docker)
- For what? Discord Bot (app)
- Actually there is ONLY local access (I should at most be able to access it via via mongo compass)
- The databases, whether postgres in background or the actual mongo, should have a username and password so that they are secured
- SSL is not planned. I use Caddy for web stuff and therefore don't have my own SSL because Caddy handles it automatically

> Debian 12 server

Can someone help with that? O.o

**Comment by AlekSi** [comment]
We use Slack for quick community discussions: https://github.com/FerretDB/FerretDB?tab=readme-ov-file#community
